.gitignore

```diff
@@ -17,6 +17,7 @@ deploy/docker-compose.env.yml
 app/web/cypress/screenshots/
 app/web/cypress/videos/
 bin/si/src/version.txt
+prelude
```

Comment:
We may update the prelude, so we don't want to ignore it.

Reply:
It doesnt ignore if its already pushed

Reply:
Nevermind. It already exists in the repo, so this is fine.

Reply:
We commented at the same time lol